src/util/psbt/mod.rs

```diff
@@ -72,6 +72,24 @@ pub struct PartiallySignedTransaction {
 }
 
 impl PartiallySignedTransaction {
+    /// Returns an iterator for the funding UTXOs of the psbt
+    /// function throws error if transaction inputs have unequal lengths
```

Comment:
It doesn't "throw" and error though. It panics. In Rust the usual style of documenting panics is like this:

```rust
/// Returns an iterator for the funding UTXOs of the psbt
///
/// For each PSBT input that contains UTXO information `Ok` is returned containing that information.
/// The order of returned items is same as the order of inputs.
///
/// ## Errors
///
/// The function returns error when UTXO information is not present or is invalid.
///
/// ## Panics
///
/// The function panics if the length of transaction inputs is not equal to the length of PSBT inputs.
```

(See below for "or invalid" case.)